MonkeyWrench.Database/DB.cs

```diff
@@ -499,7 +500,7 @@ public DBLane CloneLane (int lane_id, string new_name, bool copy_files)
 
 					foreach (DBHostLaneView hostlanemaster in master.GetHosts (this)) {
 						DBHostLane clone = new DBHostLane ();
-						clone.enabled = false;
+						clone.enabled = hostlanemaster.enabled;
```

Comment:
This means wrench will start working on the cloned lane immediately, before you've had a chance to change any values (such as max commit, which usually points to the branch you want to build).

Maybe make the disable the cloned lane (currently the master lane's value is copied)?